dns-records/hopic-sdpac.noble.phac-aspc.alpha.canada.ca.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: dns.cnrm.cloud.google.com/v1beta1
+kind: DNSRecordSet
+metadata:
+  name: "hopic-sdpac-noble-phac-aspc-alpha-canada-ca-ns"
+  namespace: "alpha-dns"
+  annotations:
+    projectName: "HoPiC"
+    sourceCodeRepository: "https://github.com/PHACDataHub/cpho-phase2"
+spec:
+  name: "hopic-sdpac.noble.phac-aspc.alpha.canada.ca."
```

Comment:
Main issue is the dot delegated domain name. Youre requesting **hopic-sdpac.noble** zone be delegated to your nameserver.

My advice would be to manage **hopic-sdpac.phac-aspc.alpha.canada.ca** and then delegate your own zone for "noble" or use noble as its own A or CNAME record.

Reply:
We already have `hopic-sdpac.phac-aspc.alpha.canada.ca` in the experimentation space and wanted to have something similar in noble experimentation so that we can stand up a self-contained copy of the project in that space. As you suggested, we can manage (which we already do) and then delegate our own zone for noble in the noble experimentation project space, however in our case, that might cause issues if our experimentation project that's hosting `hopic-sdpac.phac-aspc.alpha.canada.ca` gets deleted, right? Please let me know if I'm missing something here.

Since the main issue is dot delegated domain name, how about using `-` instead of a `.`, i.e, `hopic-sdpac-noble.phac-aspc.alpha.canada.ca`?

Reply:
sounds good, youll have to update the DNS record and nameservers; they will both change.

Reply:
fixed in [97062d3](https://github.com/PHACDataHub/dns/pull/53/commits/97062d30684260b9d259aa26a6cfbdfed9ecc6d2).